js/index.js

```diff
@@ -39,4 +39,27 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.src = siteContent["nav"]["img-src"];
+
+let ctaImg = document.getElementById("cta-img");
+ctaImg.src = siteContent["cta"]["img-src"];
+
+let midImg = document.getElementById("middle-img");
+midImg.src = siteContent["main-content"]["middle-img-src"];
+
+//Update to Navigation
+const navA = document.querySelectorAll("A");
+console.log(navA);
+
+navA[0].textContent = siteContent["nav"]["nav-item-1"];
+navA[1].textContent = siteContent["nav"]["nav-item-2"];
```

Comment:
this is fine and it works, but I would stretch yourself to see if you can loop over navA nodelist.